packages/terra-functional-testing/lib/docker/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:3.141.59-zirconium
+   ports:
+     - "4444:4444"
+
+  chrome:
+    image: selenium/node-chrome:3.141.59-zirconium
+    volumes:
+      - /dev/shm:/dev/shm
+    environment:
+      HUB_HOST: hub
+      HUB_PORT: 4444
+      TZ: 'America/Chicago'
```

Comment:
We should consider removing this. Internally this timezone used the default and running tests using the docker server compared to the selenium grid has given differing results for time components. 

- https://github.com/cerner/terra-toolkit-boneyard/issues/7

Reply:
I'll remove this for now and add a note on the epic issue to discuss this further prior to an official release.